dracut/modules.d/90kiwi-live/parse-kiwi-live.sh

```diff
@@ -13,6 +13,12 @@ fi
 [ "${liveroot%%:*}" = "live" ] || return 1
 
 modprobe -q loop
+modprobe -q aoe
```

Comment:
I am just wondering if this could cause the script to fail if the module can't be loaded. I think this is script is sourced not executed (not fully sure about that), so if it is sourced it might be executed in some context including `set -e` then this script would fail due to aoe even when `rd.kiwi.live.pxe` is not provided.

I know this might be a chain of unlikely circumstances, but in any case I think it might be just safer to `modprobe aoe` only if needed, as it was.

Reply:
I agree, will move the loading to conditional place

Reply:
done now